egs2/fsc_challenge/asr1/conf/tuning/train_asr_transformer_adam.yaml

```diff
@@ -0,0 +1,31 @@
+# network architecture
```

Comment:
Can you combine some very similar configs? like this one and the `train_asr_transformer.yaml`, `train_asr_transformer_specaug.yaml`

Reply:
Sounds good! I will try to combine some of them and remove some other configs not used.